packages/wrangler/package.json

```diff
@@ -37,6 +37,7 @@
   ],
   "dependencies": {
     "esbuild": "0.14.1",
+    "esbuild-plugin-replace": "^1.1.0",
```

Comment:
why did adding this one package delete 3.2k lines from package-lock.json, lol.

Reply:
I'm gonna sit down one afternoon and figure out how to `npm install` properly. This drives me nuts. It seems to give me different behavior every time I run it?!